lib/api/asset/v1/types.ts

```diff
@@ -61,6 +61,13 @@ export interface AssetManagement {
   managingOrganisationId: string;
   owner: string;
   isTMOManaged: boolean;
+  propertyOccupiedStatus?: string;
+  propertyOccupiedStatusReason?: string;
+  isNoRepairsMaintenance?: boolean;
+  councilTaxType?: string;
+  councilTaxLiability?: string;
+  isTemporaryAccomodation?: boolean;
+  readyToLetDate?: boolean;
```

Comment:
Are all of these required for this PR? Could the rest of these be part of a separate PR?

Reply:
They're all required, I did try just sending isCouncilProperty via Postman but the API didn't like it unfortunately